tools/coverage_unit_tests.rc

```diff
@@ -1,8 +1,17 @@
 [run]
 omit =
     */_library*
+    */nimodinst/__init__.py
+    */nimodinst/_converters.py
+    */nimodinst/_visatype.py
+    */nimodinst/errors.py
+    */nitclk/__init__.py
+    */nitclk/_attributes.py
+    */nitclk/_converters.py
+    */nitclk/_visatype.py
+    */nitclk/errors.py
 include =
     */nifake/*
     */nimodinst/session.py
-    */nitclk/*
+    */nitclk/session.py
```

Comment:
does this get the coverage 100% up?
if so, the file with the minimum coverage should be updated in tandem

Reply:
No, but it gets the coverage to 89%

We will never get to 100% without a lot more work.